projecttakeoffapp/templates/philpixel.html

```diff
@@ -12,10 +11,19 @@
 {% block main %}
     <script>PopUp('This was a test of getting Javascript to work!');</script>
 
+    <div style="margin: 0; position: absolute; top: 20%; left: 50%; transform: translate(-50%, -50%);">
```

Comment:
Perhaps consider making a class for this CSS styling in the css file ("styles.css")